RELEASE_NOTES.md

```diff
@@ -1,4 +1,6 @@
 # Release Notes
+## 1.48 9-Jun-2020
+- Replaced Travis CI/CD with Github Actions
```

Comment:
This page changed in the meantime (see master) but we also want to keep it clean until dxWDL release so either remove or add an "unreleased" note. It's better to not keep an unreleased section though since then you'd need to ask all developers to always update it when they want to merge to master to keep it up to date. Please remove.

Reply:
removed